components/sso-hostobject/org.wso2.carbon.hostobjects.sso/src/main/java/org/wso2/carbon/hostobjects/sso/internal/util/SSOAgentCarbonX509Credential.java

```diff
@@ -0,0 +1,96 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fixed in #1581 (https://github.com/wso2/carbon-apimgt/commit/0e5a93cca133738cc74a26863424e20db485892d)